src/pages/Miruro/main.ts

```diff
@@ -0,0 +1,75 @@
+import { pageInterface } from '../pageInterface';
+
+export const Miruro: pageInterface = {
+  name: 'Miruro',
+  domain: ['https://www.miruro.tv', 'https://www.miruro.online'],
+  languages: ['English'],
+  type: 'anime',
+  isSyncPage(url) {
+    return Boolean(j.$('.player[data-media-player]').length);
+  },
+  isOverviewPage(url) {
+    return false;
+  },
+  getImage() {
+    return $('.player[data-media-player] .vds-poster').attr('src');
+  },
+  sync: {
+    getTitle(url) {
+      return utils.urlPart(url, 5).replace(/-/g, ' ').trim();
```

Comment:
Using the url for title is always dangerous as it eats special characters and there are anime where the second season is only the addition of a special character

Reply:
Now i'm used the Anime text title which is shown just below the video player directly from HTML